libsolidity/lsp/LanguageServer.cpp

```diff
@@ -212,6 +213,40 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 		if (typeFailureCount)
 			m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
 	}
+
+	Json::Value jsonRemappings = _settings["remappings"];
+
+	if (jsonRemappings)
+	{
+		bool typeFailures = false;
+		if (jsonRemappings.isArray())
+		{
+			vector<frontend::ImportRemapper::Remapping> remappings;
+			for (Json::Value const& jsonPath: jsonRemappings)
+			{
+				if (
+					jsonPath.isObject() &&
+					jsonPath.get("context", "").isString() &&
+					jsonPath.get("prefix", "").isString() &&
+					jsonPath.get("target", "").isString()
+				)
+					remappings.emplace_back(ImportRemapper::Remapping{
+						jsonPath.get("context", "").asString(),
+						jsonPath.get("prefix", "").asString(),
+						jsonPath.get("target", "").asString(),
+					});
```

Comment:
```suggestion
					remappings.emplace_back(
						jsonPath.get("context", "").asString(),
						jsonPath.get("prefix", "").asString(),
						jsonPath.get("target", "").asString(),
					);
```

Reply:
By the way, these `get`s can also be replaced with `getOrDefault<string>(jsonPath["prefix"], "")`.